opencxl/cxl/component/cxl_cache_manager.py

```diff
@@ -42,11 +44,65 @@ def __init__(
         self._upstream_fifo: FifoPair
         self._cache_device_component = None
 
+        self.device_entries = {}  # maps CQID -> future host UQID
```

Comment:
This actually maps CQID to the future received H2D packet. It's just a silly comment that I can fix later.